### PR TITLE
[core] Add device id for EHL

### PR DESCRIPTION
**Pull request by Alexandr-Konovalov**
Issue: MDP-58421

### CODE REVIEW

File: api/include/mfxcommon.h [hunk 1]
```diff
@@ -174,7 +174,8 @@ enum {
     MFX_PLATFORM_CANNONLAKE     = 20,
 #endif
 #if (MFX_VERSION >= 1027)
-    MFX_PLATFORM_ICELAKE = 30,
+    MFX_PLATFORM_ICELAKE        = 30,
+    MFX_PLATFORM_ELKHARTLAKE    = 33,
```
Comment (dvrogozh) [inline api/include/mfxcommon.h]: I guess this should go to the next API version? 1.31 or 1.30 if it is not yet released. @onabiull  - ?

Comment (onabiull) [reply api/include/mfxcommon.h]: Right now we have 1.30 as latest API. So any changes should target at least 1.31, but better to omit exact version and use just NEXT.

But. I believe we should not version enum expansions. Change is too small and safe from backward compatibility point of view. 

So, please remove that guarding ifdef from this piece of code.

**Comment by Alexandr-Konovalov** [comment]
> Please put MFX_

@onabiull Could you please clarify?

File: api/include/mfxcommon.h (hunk 1 above)
Comment (Alexandr-Konovalov) [reply api/include/mfxcommon.h]: @onabiull Done.

**Pull request merged**